app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :load_user, only: :show
+
+  def show
+    @user = User.find(params[:id])
```

Comment:
dont need this line anymore
because you have load_user function, it will load @user before run action show

Reply:
Ok done